vllm_ascend/worker/model_runner_v1.py

```diff
@@ -2191,7 +2180,11 @@ def profile_run(self) -> None:
             self._dummy_run(mc2_tokens_capacity,
                             with_prefill=True,
                             is_profile=True)
+        origin_max_num_tokens = self.max_num_tokens
+        if self.pcp_size > 1:
+            self.max_num_tokens = self.max_num_tokens // self.pcp_size
         super().profile_run()
+        self.max_num_tokens = origin_max_num_tokens
```

Comment:
I'm not very understand why the code logic is so complex. Is it OK to just like:

```
if self.pcp_size > 1:
    self.max_num_tokens = self.max_num_tokens // self.pcp_size
super().profile_run()
```
The other two lines can be removed?